website/docs/patterns/multi-select-patterns/partials/specifications/specifications.md

```diff
@@ -0,0 +1,43 @@
+## Anatomy
+
+![Multi-select anatomy](/assets/patterns/multi-select-patterns/selection-anatomy.png)
+
+| Element | Usage | 
+|---------|-------|
+| Selected count | Required; displays the number of selected items in the table. |
+| Selected message | Required; explicitly communicates that results in the table have been selected. |
+| Total results  | Optional, but recommended; displays the total number of results in the data set. |
+| Actions trigger | Required; use either a `secondary` or `critical` [Button](/components/button)for single actions, or a `secondary` [Dropdown](/components/dropdown) for multiple actions. |
+| Bulk actions | Optional; consolidates multiple bulk actions into a single element. |
+| Bulk selection | Optional; consolidates multiple selection functions into a single element. |
+
+## Spacing
+
+### Multi-select pattern
+
+Within the Multi-select pattern itself, use a 16px horizontal gap between the message and the actions.
+
+![Spacing within the multi-select pattern](/assets/patterns/multi-select-patterns/multi-select-pattern-spacing.png)
+
+### In context
+
+Within a larger context, use the same spacing recommendation of 16px vertical gap between the pattern, filter elements, and data set.
+
+![Spacing in context with other elements](/assets/patterns/multi-select-patterns/multi-select-in-context-spacing.png)
+
+## Basic layout
+
+In a basic example of a Table, place the selection pattern above the top-right corner of the Table.
+
+![A basic multi-select example in context](/assets/patterns/multi-select-patterns/multi-select-in-context-basic.png)
+
+### Layout with a filter pattern
+
+When composing a Multi-select pattern with a [Filter bar](/patterns/filter-patterns?tab=specifications#filter-bar-1) and [Applied filters](/patterns/filter-patterns?tab=specifications#applied-filters-1), maintain the sample placement.
+
+![Multi-select with a Filter Pattern](/assets/patterns/multi-select-patterns/multi-select-in-context-filters.png)
```

Comment:
I intentionally scaled these up a bit; since the surrounding UI is meant to be more representative I wanted to make them stand out a bit more. I can reduce them if you feel they're a bit unrealistic